spec/webauthn/public_key_credential_with_assertion_spec.rb

```diff
@@ -233,5 +235,111 @@
         end
       end
     end
+
+    context "when verifying a migrated U2F credential" do
```

Comment:
I've just noticed that we have very similar test cases [here](https://github.com/cedarcode/webauthn-ruby/blob/dc77a68/spec/webauthn/authenticator_assertion_response_spec.rb#L306:L343)...

Maybe it's a good idea to change `spec/webauthn/public_key_credential_with_assertion_spec.rb` to mock `WebAuthn::AuthenticatorAssertionResponse#verify` so that we can just test that we are sending the right parameters to it 🤔